blazar/plugins/base.py

```diff
@@ -149,6 +149,11 @@ def get_query_options(self, params, index_type):
                       unsupported)
         return options
 
+    def is_project_allowed(self, project_id, resource):
+        if "authorized_projects" in resource:
```

Comment:
Are we sure this will always be a list?

Reply:
Ah, I see that it's doing pure string checking actually. We may want to parse this to actually enforce the comma-separated structure (just ensuring it's a string and then splitting on "," should suffice). Otherwise people could put total junk in there and they will only find out their resources are busted if somebody comes along later and makes the comma-separated claim explicit. Best to be restrictive now.